test/integration_test.go

```diff
@@ -421,7 +421,12 @@ func (ts *IntegrationTestSuite) TestChildWFWithParentClosePolicyTerminate() {
 	ts.NoError(err)
 	resp, err := ts.libClient.DescribeWorkflowExecution(context.Background(), childWorkflowID, "")
 	ts.NoError(err)
```

Comment:
probably not necessary any more?  I don't think GetWorkflow needs a runid anyway...